base/cvd/cuttlefish/host/commands/cvd/cli/commands/load_configs.cpp

```diff
@@ -167,10 +172,9 @@ class LoadConfigsCommand : public CvdCommandHandler {
         group.SetAllStates(cvd::INSTANCE_STATE_PREPARE_FAILED);
         instance_manager_.UpdateInstanceGroup(group);
       }
-      CF_EXPECTF(
-          std::move(fetch_res),
-          "Failed to fetch build artifacts, check {} for details",
-          GetFetchLogsFileName(cvd_flags.load_directories.target_directory));
+      CF_EXPECTF(std::move(fetch_res),
+                 "Failed to fetch build artifacts, check {} for details",
```

Comment:
nit: `check '{}' for details`

So that a malformed filename is more obvious.

Reply:
Done